webpack.config.js

```diff
@@ -157,15 +150,11 @@ module.exports = {
                             {
                                 pattern: /\[hash\]/g,
                                 replacement: function (match, p1, offset, string) {
-                                    const randomhash = new Date().toString();

```

Comment:
This is the cachebuster. The net effect is this before/after in the HTML:

```diff
- <script src="index.js?Mon Jun 07 2021 23:58:26 GMT+0000 (UTC)"></script>
+ <script src="index.js?1623369766512"></script>
```

Not a _necessary_ change, but.. Better.